spec/fixtures/default-invalid-missing.json

```diff
@@ -17,6 +17,7 @@
     "local>kachick/renovate-config-asdf//plugins/java.json5",
     "local>kachick/renovate-config-asdf//plugins/kotlin.json5",
     "local>kachick/renovate-config-asdf//plugins/kustomize.json5",
+    "local>kachick/renovate-config-asdf//plugins/lua.json5",
```

Comment:
I feel annoy and meaningless to update these fixtures in each PRs. So I would like to improve the linter as another task.